sso-samples/oidc-sso-sample/pickup-manager/src/main/webapp/bcIFrame.jsp

```diff
@@ -0,0 +1,29 @@
+<html>
+<head>
+    <title>Backchannel Logout Session Management RP IFrame</title>
+    <!-- JQuery -->
+    <script src="libs/jquery_3.3.1/jquery.min.js"></script>
+    <script language="JavaScript" type="text/javascript">
+
+    function check_session() {
+        $.ajax({
+            url: 'check-bc-logout',
```

Comment:
Shall we use double quotes. Check other places as well.

```suggestion
            url: "check-bc-logout",
```

Reply:
Addressed with abacc0a820911a6e100fef13d6845ae9845ac813

Reply:
Addressed with abacc0a820911a6e100fef13d6845ae9845ac813